docs/components/layout/layout.module.css

```diff
@@ -83,6 +83,7 @@
   background-color: var(--bg-secondary);
   box-shadow: inset -2px 0px 0px var(--sidebar-shadow);
   flex-grow: 1;
+  padding: 1rem 0.5rem;
```

Comment:
Skal ikke active-border være helt inntil kanten?

Reply:
Siden active box-shadow ikker er inline må vi enten ha padding rundt, eller gjøre box-shadow inline her tenker jeg
![Screenshot 2021-03-26 at 14 41 37](https://user-images.githubusercontent.com/26967723/112801472-3b911480-9071-11eb-9a7d-4a676ed5747e.png)
![Screenshot 2021-03-26 at 14 41 32](https://user-images.githubusercontent.com/26967723/112801477-3c29ab00-9071-11eb-9267-5252b77d7b90.png)

Reply:
Ah, det gir mening! Da stemmer jeg for å bruke inset outline.